base/applications/cmdutils/more/more.c

```diff
@@ -54,6 +208,13 @@ PagePrompt(PCON_PAGER Pager, DWORD Done, DWORD Total)
     HANDLE hInput = ConStreamGetOSHandle(StdIn);
     DWORD dwMode;
     KEY_EVENT_RECORD KeyEvent;
+    BOOL fCtrl;
+    DWORD nLines;
+    WCHAR ch;
+    static UINT s_nPromptID = IDS_CONTINUE_PROGRESS;
+    static WCHAR s_chSubCommand = 0;
```

Comment:
I'm not a fan of having these as static variables **within** the function.
Indeed, if paging display is cancelled early for this file and another file is going to be displayed, how are you going to reset these variables to their default values?

Reply:
OK.